### PR TITLE
[GLO-1885] Add generated sdk with UpdateSSO endpoint

### PR DESCRIPTION
**Pull request by emayukos**
Add generated files and test for UpdateSSO endpoint

@vendasta/lucky-charms

### CODE REVIEW

File: tests/PartnerClientTest.php [hunk 1]
```diff
@@ -67,4 +68,27 @@ public function testChangeSeatsHappyPath()
...
+        $req->setEnableSso(false);
+
+        try {
+            $resp = $partnerClient->UpdateSSO($req);
+        } catch (Vendasta\Vax\SDKException $e) {
+            self::assertEquals(404, $e->getCode(), 'expected a 404');
+            return;
+        }
+
+        self::assertNotEmpty($resp, 'expected a response');
```
Comment (ParkerReese) [inline tests/PartnerClientTest.php]: Shouldn't we assert the response is empty? Or how does that work with google.protobuf.Empty

**Comment by emayukos** [comment]
The test for UpdateSSO and GetDomainInformation pass but the test for ChangeSeats fails on ListSubscriptions. Not sure if this is due to my IDE not being set up correctly

File: tests/PartnerClientTest.php (hunk 1 above)
Comment (emayukos) [reply tests/PartnerClientTest.php]: Yes, not sure why my test still passed 👀

File: tests/PartnerClientTest.php [hunk 2]
```diff
@@ -65,6 +66,29 @@ public function testChangeSeatsHappyPath()
...
+        $environment = getenv("ENVIRONMENT");
+        if ($environment == null) {
+            $environment = "DEMO";
+        }
+        $partnerClient = new PartnerClient($environment);
+
+        $req = new UpdateSSORequest();
+        $req->setDomain("google.com");
+        // disable SSO
+        $req->setEnableSso(false);
```
Comment (coreyhickson) [inline tests/PartnerClientTest.php:87]: probably too late for this, but the wording around this endpoint is a bit weird `setEnableSSO` and `false` doesn't make a lot of sense

File: tests/PartnerClientTest.php [hunk 3]
```diff
@@ -65,6 +66,29 @@ public function testChangeSeatsHappyPath()
...
+
+        try {
+            $resp = $partnerClient->UpdateSSO($req);
+        } catch (Vendasta\Vax\SDKException $e) {
+            self::assertEquals(404, $e->getCode(), 'expected a 404');
```
Comment (coreyhickson) [inline tests/PartnerClientTest.php]: Imo we don't want to pass on a 404 for this test, I think this should fail the test

**Review by coreyhickson — APPROVED** [review]

File: tests/PartnerClientTest.php (hunk 2 above)
Comment (emayukos) [reply tests/PartnerClientTest.php:87]: I think the EnableSSO field itself makes sense as long as users are aware that this is a setter method for a boolean field

Comment (coreyhickson) [reply tests/PartnerClientTest.php:87]: Oh right, I forgot this was the field 👍 Ok gotcha, that makes sense I just misread that before

**Review by ParkerReese — APPROVED** [review]

**Pull request merged**